examples/csharp/azure/MyTerraformStack.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RestoreSources>$(RestoreSources);./../../../dist/dotnet;https://api.nuget.org/v3/index.json</RestoreSources>
```

Comment:
This does match how the other languages work so we can probably leave alone, but it would be kind of nice if examples didn't require it to better match using a release version.